src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
The stylesheet link is pointing to a '.scss' file. HTML cannot directly use SCSS files; they need to be compiled to CSS first. Ensure the stylesheet is linked to a '.css' file instead.